code_challenges/code_wars/highest_and_lowest.py

```diff
@@ -0,0 +1,16 @@
+# https://www.codewars.com/kata/554b4ac871d6813a03000035
+
+
+def max_and_min(digits):
+    num_list = [int(num) for num in str(digits).split()]
+    if len(num_list) < 2:
+        return f"{num_list[0]} {num_list[0]}"
+    else:
+        eval_nums = [num_list[0], num_list[1]]
+        i = 0
```

Comment:
Since the if above is a guard clause and returns early, no need for the setup to be in the else branch. Remove the else al together and pull the variable assignment back a tab.